### PR TITLE
adminActionCenter #200

### PR DESCRIPTION
**Pull request by CalvinFronda**

### CODE REVIEW

File: src/components/FeatureViews/AdminPortal/AdminPortal.js [hunk 1]
```diff
@@ -35,17 +35,17 @@ function AdminPortal(props) {
...
                 <Grid container layout={"row"}>
                     <Grid item xs={12}>
-                        <BackButton/>
-                        <hr/>
+                        <BackButton />
+                        <hr />
                     </Grid>
                     <Grid item xs={12}>
-                        <Typography variant={"h3"} align={"left"}>Admin Portal</Typography>
+                        <Typography variant={"h3"} align={"left"} style={{ paddingBottom: "20px" }}>Admin Portal</Typography>
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminPortal.js]: Be consistent and use classnames

File: src/components/FeatureViews/AdminPortal/AdminActionCenter.js [hunk 2]
```diff
@@ -66,112 +75,117 @@ function AdminActionCenter(props) {
...
+            onClose={handleClose("course")}>
+            <MenuItem
+                component={NavLink}
+                to={`/registration/form/course_details`}>
+                <ListItemText primary="NEW COURSE" />
+            </MenuItem>
+            <MenuItem
+                component={NavLink}
+                to={`/adminportal/manage-course-categories`}>
+                {/*<NewTutor className="icon innerIcon" />*/}
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminActionCenter.js]: Delete comments

File: src/components/FeatureViews/AdminPortal/AdminActionCenter.js [hunk 3]
```diff
@@ -66,112 +75,117 @@ function AdminActionCenter(props) {
...
-                        className="button"
-                        aria-controls="simple-menu"
-                        aria-haspopup="true"
-                        onClick={handleClick("tuition")}>
-                        Manage Tuition
+        </Grid>
+        <Grid item>
+            <Button
+                variant="outlined"
+                // color="primary"
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminActionCenter.js]: Delete comments

File: src/components/FeatureViews/AdminPortal/AdminActionCenter.js [hunk 4]
```diff
@@ -66,112 +75,117 @@ function AdminActionCenter(props) {
...
+                // color={"primary"}
+                className={classes.setParent}>
+                <AddPerson className="icon" />
+                Add Instructor
+            </Button>
+        </Grid>
+        <Grid item>
+            <Button
+                variant="outlined"
+                // color="primary"
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminActionCenter.js]: Delete comments

File: src/components/FeatureViews/AdminPortal/AdminActionCenter.js [hunk 5]
```diff
@@ -66,112 +75,117 @@ function AdminActionCenter(props) {
...
-                        aria-controls="simple-menu"
-                        aria-haspopup="true"
-                        onClick={handleClick("course")}>
-                        Manage Course
+    }, [location]);
+
+    return (<Grid container spacing={16} className="registration-buttons">
+        <Grid item>
+            <Button component={NavLinkNoDup} to={"/registration/form/instructor"}
+                // color={"primary"}
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminActionCenter.js]: Delete comments

File: src/components/FeatureViews/AdminPortal/AdminActionCenter.js [hunk 6]
```diff
@@ -1,25 +1,48 @@
...
+import AssignmentIcon from "@material-ui/core/SvgIcon/SvgIcon";
 import MenuItem from "@material-ui/core/MenuItem";
 import ListItemText from "@material-ui/core/ListItemText";
+import PinDropIcon from '@material-ui/icons/PinDrop';
 import Menu from "@material-ui/core/Menu";
+import AddPerson from "@material-ui/icons/PersonAdd";
+import EventNoteIcon from '@material-ui/icons/EventNote';
+
+
+const useStyles = makeStyles(theme => ({
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminActionCenter.js]: Delete. this variable is never used.

File: src/components/FeatureViews/AdminPortal/AdminPortal.js [hunk 7]
```diff
@@ -6,33 +6,49 @@ import Grid from "@material-ui/core/Grid";
...
+    setParent: {
+        backgroundColor: "#39A1C2",
+        color: "white",
+        // padding: "",
+    }
+})
+
+function AdminPortal(props) {
+    const [anchorEl, setAnchorEl] = useState(null);
+
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminPortal.js]: Delete extra lines

File: src/components/FeatureViews/AdminPortal/AdminPortal.js [hunk 8]
```diff
@@ -6,33 +6,49 @@ import Grid from "@material-ui/core/Grid";
...
-import {withRouter} from "react-router-dom";
+import { withRouter } from "react-router-dom";
+import Checkbox from "@material-ui/core/Checkbox";
 
 import BackButton from "../../BackButton";
 import AdminActionCenter from "./AdminActionCenter";
 import AdminViewsRoutes from "../../Routes/AdminViewsRoutes";
 
 
+const useStyles = makeStyles({
```
Comment (dhuangg) [inline src/components/FeatureViews/AdminPortal/AdminPortal.js]: Delete unused variables

**Comment by NeilAllavarpu** [comment]
What's the status on this?

**Comment by dhuangg** [comment]
this should be closed. I already built this on my own

**Pull request closed without merging**